LEAF_Nexus/sources/System.php

```diff
@@ -242,4 +256,90 @@ public function refreshOrgchartEmployees()
 
         return $ret;
     }
+
+    /**
+     * Get primary admin.
+     *
+     * @return array|false array with primary admin's info
+     */
+    public function getPrimaryAdmin()
```

Comment:
Please add return type.  public function getPrimaryAdmin(): string

You'll need to change the else return to ""

Reply:
added return type

Reply:
Also changed the returns so it stays standard array